### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by philip-khor**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by philip-khor** [comment]
@conda-forge-admin, please rerender

**Comment by philip-khor** [comment]
@conda-forge/optuna ready for review, thanks!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -23,12 +23,13 @@ requirements:
   run:
     - alembic
     - cliff
-    - cmaes
+    - cmaes >=0.5.1
     - colorlog
     - joblib
     - numpy
+    - packaging
```
Comment (hvy) [inline recipe/meta.yaml:30]: A recent change in our `setup.py` https://github.com/optuna/optuna/pull/1599. How about incorporating this as well?
```suggestion
    - packaging >=20.0
```

**Comment by philip-khor** [comment]
@conda-forge-admin, please rerender

**Review by hvy — APPROVED** [review]
LGTM and consistent with `setup.py`, thanks.

**Review by HideakiImamura — APPROVED** [review]
LGTM.

**Pull request merged**